packages/azure/README.md

```diff
@@ -0,0 +1,114 @@
+validator network. You must merge them into a single JSON
+`template.json` file before deploying:
+
+```
+npm run build
+```
+
+## Usage
+
+Pick a location and create a resource group:
+
+```
+LOCATION=... # e.g., westus2
+GROUP=... # e.g., celoValidator
+az group create \
+   --name $GROUP --location $LOCATION
+```
+
+Deploy `template.json`, specifying all the required parameters:
+
+```
+az group deployment create \
+   --resource-group $GROUP \
+   --name ${GROUP}-deployment \
+   --template-file=template.json \
+   --parameters \
+   userObjectId=... \
+   proxyName=... \
+   validatorName=... \
+   adminUsername=... \
+   adminPublicKey=... \
+   attesterPostgreSQLUsername=... \
+   attesterPostgreSQLPassword=... \
+   validatorAccountAddress=... \
+   proxyPublicKey=... \
+   proxyPrivateKey=... \
+   validatorPrivateKey=... \
+   validatorGethAccountSecret=... \
+   attesterAccountAddress=... \
+   attesterPrivateKey=... \
+   attesterGethAccountSecret=... \
+   attesterTwilioAccountSID=... \
+   attesterTwilioAuthToken=... \
+   attesterTwilioMessageServiceSID=... \
+```
+
+You can get the value of your `userObjectId` via the Portal or `az ad user list`.
+
+You can delete all the services you created with:
+
+```
+az group delete --name $GROUP
+```
+
+### Required Parameters
+
+| Parameter | Type | Description |
+| --------- | -----| ----------- |
+| `userObjectId` | string | The object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. |
+| `proxyName` | string | The proxy name for ethstats |
+| `validatorName` | string | The validator name for ethstats |
+| `adminUsername` | string | Admin username for SSH access |
+| `adminPublicKey` | secureString | Admin public key for SSH access |
+| `attesterPostgreSQLUsername` | string | Admin username for Attester PostgreSQL server |
+| `attesterPostgreSQLPassword` | secureString | Admin password for Attester PostgreSQL server |
+| `validatorAccountAddress` | string | Authorized validator signer etherbase account address |
+| `proxyPublicKey` | string | Proxy enode public key |
+| `proxyPrivateKey` | secureString | Proxy etherbase account private key |
+| `validatorPrivateKey` | secureString | Authorized validator signer etherbase account private key |
+| `validatorGethAccountSecret` | secureString | Password to secure validatorPrivateKey |
+| `attesterAccountAddress` | string | Attester account address |
+| `attesterPrivateKey` | secureString | Attester account private key |
+| `attesterGethAccountSecret` | secureString | Password to secure attesterPrivateKey |
+| `attesterTwilioAccountSID` | string | Attester Twilio account SID |
+| `attesterTwilioAuthToken` | secureString | Attester Twilio authentication token |
+| `attesterTwilioMessageServiceSID` | string | Attester Twilio message service SID |
+
+### SSH access
+
+The proxy and attester expose port 22 on their public IP addresses.
```

Comment:
It's best practice to leave these locked down by default, but to create an explicit "Deny" policy to make it easy for the user to open after the deployment.